croud/clusters/commands.py

```diff
@@ -159,12 +160,58 @@ def clusters_scale(args: Namespace) -> None:
     )
 
 
-def import_jobs_create(args: Namespace) -> None:
-    body = {
-        "type": args.type,
+def import_jobs_create_from_url(args: Namespace) -> None:
+    extra_body = {
         "url": {
             "url": args.url,
-        },
+        }
+    }
+    args.type = "url"
+    import_jobs_create(args, extra_payload=extra_body)
+
+
+def _get_org_id_from_cluster_id(client, cluster_id: str) -> Optional[str]:
+    data, errors = client.get(f"/api/v2/clusters/{cluster_id}/")
+    if errors or not data:
+        return None
+
+    project_id = data["project_id"]
+
+    data, errors = client.get(f"/api/v2/projects/{project_id}/")
+    if errors or not data:
+        return None
+
+    return data["organization_id"]
+
+
+def import_jobs_create_from_file(args: Namespace) -> None:
+    file_id = args.file_id
+
+    if args.file_path:
+        client = Client.from_args(args)
+        org_id = _get_org_id_from_cluster_id(client, args.cluster_id)
+        if not org_id:
+            print_error("Could not find the organization related to the cluster.")
+            return
+
+        data, errors = op_upload_file_to_org(client, org_id, args.file_path)
+        if errors or not data:
+            print_error("Aborted import job creation.")
+            return
+        file_id = data["id"]
+
+    extra_body = {
+        "uploaded_file": {
```

Comment:
```suggestion
        "file": {
```
Thomas just changed it.